nir/src/main/scala/scala/scalanative/nir/serialization/Tags.scala

```diff
@@ -209,27 +212,28 @@ object Tags {
 
   final val Val = Type + 32
 
-  final val TrueVal        = 1 + Val
-  final val FalseVal       = 1 + TrueVal
-  final val NullVal        = 1 + FalseVal
-  final val ZeroVal        = 1 + NullVal
-  final val CharVal        = 1 + ZeroVal
-  final val ByteVal        = 1 + CharVal
-  final val ShortVal       = 1 + ByteVal
-  final val IntVal         = 1 + ShortVal
-  final val LongVal        = 1 + IntVal
-  final val FloatVal       = 1 + LongVal
-  final val DoubleVal      = 1 + FloatVal
-  final val StructValueVal = 1 + DoubleVal
-  final val ArrayValueVal  = 1 + StructValueVal
-  final val CharsVal       = 1 + ArrayValueVal
-  final val LocalVal       = 1 + CharsVal
-  final val GlobalVal      = 1 + LocalVal
-  final val UnitVal        = 1 + GlobalVal
-  final val ConstVal       = 1 + UnitVal
-  final val StringVal      = 1 + ConstVal
-  final val VirtualVal     = 1 + StringVal
-  final val ClassOfVal     = 1 + VirtualVal
-
-  final val LinktimeConditionVal = 1 + ClassOfVal
+  final val TrueVal              = 1 + Val
+  final val FalseVal             = 1 + TrueVal
+  final val NullVal              = 1 + FalseVal
+  final val ZeroVal              = 1 + NullVal
+  final val SizeVal              = 1 + ZeroVal
```

Comment:
Same here, put new Tags after last definition of old ones with the same block.  There is also one more case in Types